lib/bolt/cli.rb

```diff
@@ -196,7 +197,7 @@ def parse_command
 
       # Default to verbose for everything except plans
       unless options.key?(:verbose)
-        options[:verbose] = options[:subcommand] != 'plan'
+        options[:verbose] = !%w[lookup plan].include?(options[:subcommand])
```

Comment:
Does `verbose` actually impact the output? If not I think we can omit this, we don't exclude it for other commands that don't use `verbose`.

Reply:
Nope. Removed.